app/src/main/res/values/strings.xml

```diff
@@ -35,7 +35,10 @@
     <string name="session_header_divider" translatable="false">" / "</string>
     <string name="session_load_retry">Retry</string>
     <string name="session_finished">Finished Session</string>
+    <string name="session_go_to_questionnaire">Go To Questionnaire</string>
```

Comment:
Now session_go_to_feedback become session_go_to_feedback.
So probably we can delete it.

Reply:
Sure thing!